lib/screens/department_screen/widgets/faculty_details_section.dart

```diff
@@ -1,15 +1,353 @@
+                ),
+              ],
+            ),
+          ),
+        ),
+      ],
+    );
+  }
+}
+
+class ExpandedFacultyCard extends StatelessWidget {
+  const ExpandedFacultyCard({
+    Key? key,
+    required this.name,
+    required this.designation,
+    required this.email,
+    required this.experience,
+    required this.imageUrl,
+    required this.phdGuide,
+    required this.qualification,
+    required this.specialization,
+  }) : super(key: key);
+
+  final String name;
+  final String designation;
+  final String email;
+  final String experience;
+  final String imageUrl;
+  final String phdGuide;
+  final String qualification;
+  final String specialization;
+
+  @override
+  Widget build(BuildContext context) {
+    Size size = MediaQuery.of(context).size;
+    return Column(
+      children: [
+        CollapsedFacultyCard(
+          name: name,
+          designation: designation,
+          imageUrl: imageUrl,
+        ),
+        const SizedBox(
+          height: 8,
+        ),
+        Padding(
+          padding: const EdgeInsets.symmetric(horizontal: 15),
+          child: Row(
+            children: [
+              const Padding(
+                padding: EdgeInsets.only(top: 20),
+                child: Icon(
+                  Icons.email,
+                  size: 25,
+                  color: kLightModeLightBlue,
+                ),
+              ),
+              const SizedBox(
+                width: 30,
+              ),
+              Expanded(
+                child: Wrap(
+                  spacing: 5,
+                  runSpacing: 5,
+                  children: [
+                    IconWithChipText(
+                      assetPath: 'assets/images/icons/experience.png',
+                      text: experience,
+                    ),
+                    IconWithChipText(
+                      assetPath: 'assets/images/icons/qualifications.png',
+                      text: qualification,
+                    ),
+                  ],
+                ),
+              ),
+            ],
+          ),
+        ),
+        const SizedBox(
+          height: 10,
+        ),
+        Text(
+          "Area of Specialization",
+          style: Theme.of(context)
+              .textTheme
+              .headline3!
+              .copyWith(fontWeight: FontWeight.w600, fontSize: 17),
+        ),
+        const SizedBox(
+          height: 3,
+        ),
+        ChipStyledText(
+          text: specialization,
+          width: 0.5 * size.width,
+        ),
+      ],
+    );
+  }
+}
+
+class ChipStyledText extends StatelessWidget {
+  final String text;
+  final double width;
+  const ChipStyledText({
+    Key? key,
+    required this.text,
+    this.width = 0.0,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      padding: const EdgeInsets.all(8),
+      width: width > 0 ? width : null,
```

Comment:
make width nullable. So either it will be a null or a value you don't need to make a conditional statement here